src/index.html

```diff
@@ -11,7 +11,171 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+      onsubmit="onSubmit()"
+    >
+
+      <fieldset class="form_box">
+        <legend>Personal information</legend>
+
+        <div class="formfield">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+          >
+        </div>
+        <div class="formfield">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+          >
+        </div>
+        <div class="formfield">
+          <label for="age">How old are you?
+          </label>
+          <input
+            type="number"
+            name="age"
+            id="age"
+            min="12" max="100"
+            value="12"
+          >
+        </div>
+        <div class="formfield">
+          <label for="date_of_birth">Full date of birth:</label>
+          <input
+            type="date"
+            name="date_of_birth"
+            id="date_of_birth"
+          >
+        </div>
+        <div class="formfield">
+          <label for="term">I accept the term of the afreement</label>
+          <input
+            type="checkbox"
+            name="term"
+            id="term"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form_box">
+        <legend>Registration</legend>
+
+        <div class="formfield">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+        <div class="formfield">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="5" maxlength="32"
```

Comment:
3rd item of [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md), make sure to fix it everywhere
```suggestion
            minlength="5" 
            maxlength="32"
```